org-formation/090-systems-manager/Scheduled-Script-Automation.yaml

```diff
@@ -0,0 +1,297 @@
+                      {
+                          'Accounts': TargetAccountsArray,
+                          'Regions': TargetRegionIdsArray,
+                          'TargetLocationMaxConcurrency': f'{TargetLocationMaxConcurrency}',
+                          'TargetLocationMaxErrors': f'{TargetLocationMaxErrors}',
+                          'ExecutionRoleName': f'{ExecutionRoleName}'
+                      }
+                  ]
+              )
+              print(response)
+      Environment:
+        Variables:
+            # Fn::EnumTargetAccounts is specific to org-formation. It returns all ScriptExecutionMemberBinding account IDs defined in _tasks.yaml
+            TargetAccounts:
+              Fn::Join:
+                - ","
+                - - Fn::EnumTargetAccounts ScriptExecutionMemberBinding ${account}
+            TargetRegionIds: !Ref TargetRegionIds
+            TargetLocationMaxConcurrency: !Ref TargetLocationMaxConcurrency
+            TargetLocationMaxErrors: !Ref TargetLocationMaxErrors
+            ExecutionRoleName: !Ref ExecutionRoleName
+            MasterAccountID: !Sub ${AWS::AccountId}
+            AutomationDocumentScriptExecution: !Ref AutomationDocumentScriptExecution
+            ScriptUrl: !Ref ScriptUrl
+            ScriptParameters: !Ref ScriptParameters
+            TargetTagName: !Ref TargetTagName
+            TargetTagValue: !Ref TargetTagValue
+      FunctionName: MultiAccountScriptExecution
+      Handler: index.handler
+      Role: !GetAtt AWSLambdaSSMMultiAccountScriptExecutionRole.Arn
+      Runtime: python3.12
+  AutomationAdministrationServiceRole:
+    Type: AWS::IAM::Role
+    Condition: CreateAutomationAdministrationRoleCondition
+    Properties:
+      RoleName: AWS-SystemsManager-AutomationAdministrationRole
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service: ssm.amazonaws.com
+          Action:
+          - sts:AssumeRole
+      Path: "/"
+      Policies:
+      - PolicyName: AssumeRole-AWSSystemsManagerAutomationExecutionRole
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+            - sts:AssumeRole
+            Resource:
+              Fn::Sub: arn:${AWS::Partition}:iam::*:role/AWS-SystemsManager-AutomationExecutionRole
+          - Effect: Allow
+            Action:
+            - organizations:ListAccountsForParent
+            Resource:
+            - "*"
+  AutomationDocumentScriptExecution:
+    Type: "AWS::SSM::Document"
+    Properties:
+      DocumentType: "Command"
+      Name: "Automation-Run-Script"
+      Content:
+        schemaVersion: "2.2"
+        description: "Run a parameterized script on EC2 instances"
+        parameters:
+          ScriptUrl:
+            type: String
+            description: "The URL of the script to run"
+          SsmParameterNameToEnvvarMap:
```

Comment:
hmm, i don't see the connection from `ScriptParameters` to `SsmParameterNameToEnvvarMap`.   I mean `runCommand` processes vars in SsmParameterNameToEnvvarMap but how does the passed in vars in ScriptParameters get passed to SsmParameterNameToEnvvarMap?

Reply:
Good catch:  On line 193
                      'ScriptParameters' : [ScriptParameters]
should be
                      'SsmParameterNameToEnvvarMap' : [ScriptParameters]
It's now updated in the PR.